packages/design-system/src/images/usa-banner-flag.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="16" height="11" class="ds-c-usa-banner__header-flag" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I dont think we should be applying these classes directly to the source svg file. You should be able to pass in the class when you use the SVG in React

You can look at [react-svg-loader](https://github.com/boopathi/react-svg-loader/tree/master/packages/react-svg-loader) as an example, it works similarly to our svg babel plugin

Reply:
Another small thing, could you use [svgo](https://github.com/svg/svgo/) to clean up these svg files? svgo is a svg minimizer that will remove unnecessary attributes, and it also has a `pretty` option to format them as well. I believe you can just run it via command line

Reply:
I tried to use svgo and it messed up the SVGs even when I'm only using the `--pretty` command

Reply:
Hm how did it mess them up?

Reply:
The no longer rendered - I can add a screenshot

Reply:
This is after running `svgo -f packages/design-system/src/images --pretty` which worked and updated all the SVGs

However, it broke the rendering

![Screen Shot 2020-11-09 at 3 23 39 PM](https://user-images.githubusercontent.com/1449852/98592319-8e735680-229f-11eb-8d7e-1a5df65afa3a.png)

Here is the before and after `--pretty` updates which removed the `fill="#000000"` and some dashes in the path element

### Before
```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 52 64">
    <path fill="#000000" fill-rule="evenodd" d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z"/>
</svg>
```
### After
```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 52 64">
    <path fill-rule="evenodd" d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 014 4v28a4 4 0 01-4 4H4a4 4 0 01-4-4V32a4 4 0 014-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z"/>
</svg>
```